src/zkp/pifac.rs

```diff
@@ -90,58 +122,71 @@ impl Proof for PiFacProof {
         rng: &mut R,
     ) -> Result<Self> {
         // Small names for scaling factors in our ranges
-        let sqrt_N0 = &sqrt(&input.N0);
-
-        let alpha = random_plusminus_scaled(rng, ELL + EPSILON, sqrt_N0);
-        let beta = random_plusminus_scaled(rng, ELL + EPSILON, sqrt_N0);
-
-        let sigma = input
-            .setup_params
-            .scheme()
-            .commitment_randomness(ELL, &input.N0, rng);
-
-        let (P, mu) = input.setup_params.scheme().commit(&secret.p, ELL, rng);
-        let (Q, nu) = input.setup_params.scheme().commit(&secret.q, ELL, rng);
-        let (A, x) = input
-            .setup_params
-            .scheme()
-            .commit(&alpha, ELL + EPSILON, rng);
-        let (B, y) = input
-            .setup_params
-            .scheme()
-            .commit(&beta, ELL + EPSILON, rng);
-        let (T, r) = input.setup_params.scheme().commit_with_commitment(
-            &Q,
-            &alpha,
+        let sqrt_N0 = &sqrt(&input.modulus);
+
+        let p_mask = random_plusminus_scaled(rng, ELL + EPSILON, sqrt_N0);
+        let q_mask = random_plusminus_scaled(rng, ELL + EPSILON, sqrt_N0);
```

Comment:
suggestion: you can add an inline comment that these are `alpha` and `beta` in the paper for clarity.